src/shared/alterationFiltering/SettingsMenu.tsx

```diff
@@ -160,11 +170,19 @@ export default class SettingsMenu extends React.Component<
                                         }
                                         onClick={this.onInputClick}
                                     />{' '}
-                                    Exclude germline mutations
+                                    Show only somatic
+                                    <InfoIcon
+                                        style={{ color: 'grey' }}
+                                        divStyle={{
+                                            display: 'inline-block',
+                                            marginLeft: 6,
+                                        }}
+                                        tooltip={<>Exclude germline samples</>}
```

Comment:
```suggestion
                                        tooltip={<>Exclude germline mutations</>}
```
Samples aren't excluded. Nr of columns remain the same. It's just that germline mutations for that sample aren't shown in the oncoprint

Reply:
thanks, fixed in [64c97e8](https://github.com/cBioPortal/cbioportal-frontend/pull/4709/commits/64c97e80d2d523ae9c9fefbd72df24789e1ff78e) !